utils/dockerd/files/dockerd.init

```diff
@@ -185,6 +185,9 @@ process_config() {
 	config_get ip globals ip ""
 	config_get fixed_cidr globals fixed_cidr ""
 	config_get fixed_cidr_v6 globals fixed_cidr_v6 ""
+	config_get http_proxy proxies http_proxy "${http_proxy}"
+	config_get https_proxy proxies https_proxy "${https_proxy}"
+	config_get no_proxy proxies no_proxy "${no_proxy}"
```

Comment:
If I recall correctly, isn't the last argument the default?  
this looks like `"${http_proxy}"` is the same as `""` with a couple of extra steps if I'm not mistaken.
in which case maybe we should simplify this.

Reply:
yes, the latst argument is the default value. Just use the `*_proxy` environment variables as the default value.
For example, if the `*_proxy` environment variables are already defined in the `/etc/profile`, dockerd can used them as default. If the dockerd needs different proxy, then we can configure it in the `/etc/config/dockerd`

Reply:
Oh, cool, I'm just used to seeing environment variables in uppercase, my mistake.